README.rst

```diff
@@ -116,7 +74,6 @@ Install dependencies with conda
 .. code-block:: bash 
 
     conda install pandas
-    conda install ipywidgets
```

Comment:
@annapasca @dmalt I removed this. I think we discussed on hangouts that it is not used anywhere, right?

Reply:
yes, for me it's ok! 
If someone wants to inspect the ICA by notebook, he'll do it by himself.
What do u think @dmalt ?

Reply:
oh sorry I didn't know that. Do you have the link for the notebook? So I can see what it is doing ...

Reply:
https://neuropycon.github.io/neuropycon_doc/examples/preproc_example.html#preproc-example

at the bottom :)